test/unit/remote_control_test.lua

```diff
@@ -1,3 +1,5 @@
+require('strict').off()
```

Comment:
Luatest loads all files before running tests. By this line you disable strict mode everywhere in luatest

Reply:
Yes. I know.

Do you propose to do similar to?
```lua
g.test = function()
strict.off()
-- code ...
strict.on()
end
```